packages/storybook/stories/VideoGallery/snippets/PPTLiveSharingFromViewer.snippet.tsx

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { VideoGallery } from '@azure/communication-react';
+import { Stack } from '@fluentui/react';
+import React from 'react';
+
+const MockLocalParticipant = {
+  userId: 'user1',
+  displayName: 'You',
+  state: 'Connected',
+  isMuted: true
+};
+
+// Creating a mock PowerPoint live stream element for demonstration purposes.
+const mockPptLiveElement = document.createElement('div');
+mockPptLiveElement.style.width = decodeURIComponent('100%25');
+mockPptLiveElement.style.height = decodeURIComponent('100%25');
+mockPptLiveElement.style.textAlign = 'center';
+const imageElement = document.createElement('img');
+imageElement.src = 'images/pptlive-example.png';
+imageElement.style.maxWidth = decodeURIComponent('100%25');
+imageElement.style.maxHeight = decodeURIComponent('100%25');
+mockPptLiveElement.appendChild(imageElement);
```

Comment:
Can we copy this part to somewhere else and import the element? It feels not 100% related to the snippet